hw/top_earlgrey/data/chip_conn_testplan.hjson

```diff
@@ -0,0 +1,776 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "chip_conn"
+
+  testpoints: [
+
+    /////////////////////////
+    // AON_TIMER resets    //
+    /////////////////////////
+    {
+      name: AON_TIMER_RST
+      desc: ''Verify rstmgr's resets_o is connected to aon_timer's reset port.''
+      milestone: V1
+      tests: ["aon_timer_rst.csv"]
+    }
```

Comment:
We could add `tags: ["conn"]` to each of these and import into chip testplan, so that everything is in one place.

Reply:
That is great, thanks. Will add that.